### ISSUE TITLE
docker-formula on centos 7

### ISSUE THREAD

**Issue by chinakevinguo** [saltstack-formulas/docker-formula#89]
## when i run docker-formula on centos7 ,i have below information,please tell me why,thx

```
      ID: docker package repository
Function: pkgrepo.managed
    Name: deb https://apt.dockerproject.org/repo centos-CentOS Linux 7 (Core) main
  Result: False
 Comment: Failed to configure repo 'deb https://apt.dockerproject.org/repo centos-CentOS Linux 7 (Core) main': The repo does not exist and needs to be created, but either a baseurl or a mirrorlist needs to be given
 Started: 19:45:46.771018
Duration: 12.63 ms
 Changes:   
```

**Comment by gravyboat** [comment]
Looks like something from here (https://github.com/saltstack-formulas/docker-formula/blob/master/docker/init.sls#L7) didn't get installed, You should re-run the state with verbose output and review it to determine what isn't getting installed.

**Comment by chinakevinguo** [comment]
The default repository is http://http.debian.net/debian ...maybe this the reason ???

**Comment by blbradley** [comment]
Was CentOS ever supported in this formula?

**Comment by ghostsquad** [comment]
centos uses `yum` not apt, so it sounds like the "default" is debian, and thus Centos is not supported right now.

**Comment by hoonetorg** [comment]
wouldn't it be better to make that an extra sls?
let people decide if and which repo they want to use.
on centos docker can be installed without any repo hassles
also there exist multiple formulas for defining repos like https://github.com/hoonetorg/salt-repoconf-formula which support yum and apt. 
also enterprises often use local repos which are snapshots of a specific date of the official repos.

**Comment by blbradley** [comment]
Closed by #105.
